script/build.ts

```diff
@@ -171,7 +188,15 @@ function packageApp() {
     appBundleId: getBundleID(),
     appCategoryType: 'public.app-category.developer-tools',
     darwinDarkModeSupport: true,
-    osxSign: true,
+    osxSign: {
+      // @ts-ignore (the library's type definition is wrong)
+      hardenedRuntime: true,
+      identity: true,
```

Comment:
If I'm reading things correctly I _think_ we can drop this and rely on the default behavior rather than fight the type system. It seems that by setting this to true we're causing `electron-packager` to reset it to `null`:

https://github.com/electron/electron-packager/blob/fd237af51aefe3dbbdc3a84ed380eb468317db3b/mac.js#L367-L369

This gets sent to `electron-osx-sign` which does a truthy check for `.identity` and treats anything falsy (such as `null`) as asking for an automatic detection of identity.

https://github.com/electron/electron-osx-sign/blob/0aa699d20754c9b9fe076723871ca19aa6a255e4/sign.js#L233

Reply:
i kept `identity: true` in hopes that it would signal similarly to `osxSign: true` from before. but i'm happy to remove it! (eb00ee4)